source/documentation/upgrade_guide/topics/installing_virtual_product_hypervisors_from_local_repository.adoc

```diff
@@ -89,13 +89,13 @@ Output similar to the following is displayed:
 +
 [options="nowrap" subs="normal"]
 ----
-# /home/test/rhvh-reposync
+/home/test/rhvh-reposync
 ----
 .. Run the `reposync` command:
 +
 [options="nowrap" subs="normal"]
 ----
-# reposync --repo rhvh-4-for-rhel-8-x86_64-rpms
+# repkosync --repo rhvh-4-for-rhel-8-x86_64-rpms
```

Comment:
reposync is correct word I dont know why "k" is here

Reply:
fixed